management-account/terraform/organizations-policy-service-control.tf

```diff
@@ -0,0 +1,464 @@
+    }
+
+    condition {
+      test     = "StringLike"
+      variable = "aws:PrincipalArn"
+      values   = ["arn:aws:iam::*:root"]
+    }
+  }
+}
+
+########################################
+# Modernisation Platform Member OU SCP #
+########################################
+resource "aws_organizations_policy" "modernisation_platform_member_ou_scp" {
+  name        = "Modernisation Platform Member OU SCP"
+  description = "Restricts permissions for all OUs and accounts under the Modernisation Platform Member OU"
+  type        = "SERVICE_CONTROL_POLICY"
+
+  tags = {
+    business-unit = "Platforms"
+    component     = "SERVICE_CONTROL_POLICY"
+    source-code   = join("", [local.github_repository, "/terraform/organizations-service-control-policies.tf"])
+  }
+
+  content = data.aws_iam_policy_document.modernisation_platform_member_ou_scp.json
+}
+
+data "aws_iam_policy_document" "modernisation_platform_member_ou_scp" {
+  statement {
+    effect = "Deny"
+    actions = [
+      "ec2:CreateVpcPeeringConnection",
+      "ec2:CreateVpc",
+      "ec2:CreateSubnet"
+    ]
+    resources = ["*"]
+  }
+}
+
+#######################################################
+# Modernisation Platform Member OU SCP - Prevent Root #
+#######################################################
+resource "aws_organizations_policy" "modernisation_platform_member_ou_scp_prevent_root" {
+  name        = "Modernisation Platform Member OU SCP - Prevent Root"
+  description = "Prevents root user access for all OUs and accounts under the Modernisation Platform Member OU"
+  type        = "SERVICE_CONTROL_POLICY"
+
+  tags = {
+    business-unit = "Platforms"
+    component     = "SERVICE_CONTROL_POLICY"
+    source-code   = join("", [local.github_repository, "/terraform/organizations-service-control-policies.tf"])
+  }
+
+  content = data.aws_iam_policy_document.modernisation_platform_member_ou_scp_prevent_root.json
+}
+
+data "aws_iam_policy_document" "modernisation_platform_member_ou_scp_prevent_root" {
+  statement {
+    effect    = "Deny"
+    actions   = ["*"]
+    resources = ["*"]
+
+    condition {
+      test     = "ForAnyValue:StringLike"
+      variable = "aws:PrincipalOrgPaths"
+      values   = ["${aws_organizations_organization.default.id}/*/${aws_organizations_organizational_unit.platforms_and_architecture_modernisation_platform.id}/*"]
+    }
+
+    condition {
+      test     = "StringLike"
+      variable = "aws:PrincipalArn"
+      values   = ["arn:aws:iam::*:root"]
+    }
+  }
+}
+
+####################################################
+# Modernisation Platform Member Unrestriced OU SCP #
+####################################################
+resource "aws_organizations_policy" "modernisation_platform_member_unrestriced_ou_scp" {
+  name        = "Modernisation Platform Member Unrestriced OU SCP"
```

Comment:
This _is_ a spelling mistake, but this policy is going to be deleted. This PR just brings it into Terraform management, so Terraform can do the wonders of deleting it.